js/app.js

```diff
@@ -0,0 +1,30 @@
+$(function(){
+  $("#stopButton").click(illuminateRed);
+  $("#slowButton").click(illuminateYellow);
+  $("#goButton").click(illuminateGreen);
+
+ function illuminateRed() {
+   console.log ("this is working");
```

Comment:
@chloeeffron good job using `console.log()` to help with your development and debugging. Just remember to remove this once you know its working and especially when you push your code.